### ISSUE TITLE
rename master branch to main

### ISSUE THREAD

**Issue by jakkdl** [HypothesisWorks/hypothesis#4108]
It's breaking my local QoL scripts :(

**Comment by Zac-HD** [comment]
Ugh, I guess with `git` having changed the default for new repos it'd be better to have changed, but there's a bunch of annoying migration work that nobody wants to do.  Some of that requires admin permissions i.e. I have to do it, but if you wanted to replace all the in-repo references to `master` with `main` that'd help a lot...

**Comment by jakkdl** [comment]
yeah and the general github ecosystem has quite consistently transitioned, the large majority of repositories I interact with have switched to main/

**Comment by skirpichev** [comment]
Here is my recipe for poor people (like me) to workaround persons, doing master->main renaming in upstream repos:
```
git symbolic-ref refs/heads/master refs/heads/main
git symbolic-ref refs/remotes/origin/master refs/remotes/origin/main
```

Sure, similar works if you prefer main over master locally.

**Comment by Zac-HD** [comment]
closing per https://github.com/HypothesisWorks/hypothesis/pull/4109#issuecomment-3515361582
